epics/ca.py

```diff
@@ -1031,13 +1031,15 @@ def array_cast(data, count, ntype, use_numpy):
             else:
                 out = numpy.ctypeslib.as_array(memcopy(data))
         else:
-            out = memcopy(data)
+            out = list(data)
```

Comment:
Why replace memcopy() (which should be copy.copy()) with list() here? 

Does this have anything to do with zero-length strings?  What else with this change break?